coralproject/talk
# Site wide comments

## What does this PR do?

Adds a unique comment stream for each URL.

## How do I test this PR?

1. Visit `http://localhost:3000`
2. `http://localhost:3000/admin/embed/stream/preview`
3. There shuld be two seperately loading comment streams.
4. VIsit `http://localhost:3000?stuff=things`. You should see the same comment stream as `http://localhost:3000`
5. If you can figure out how to, visit `https://localhost:3000`. You should see the same comment stream as `http://localhost:3000`

## Note
Went ahead and added a bunch of missing asset model tests in this PR.

@coralproject/tech
